juju-charms/charm-trilio-data-mover/src/files/trilio/trilio_sudoers

```diff
@@ -1 +0,0 @@
-nova ALL = (root) NOPASSWD: /home/tvault/.virtenv/bin/privsep-helper *
```

Comment:
@abhijeetpatra Are we removing this?

Reply:
yes, when s3 package would be provided, the sudoers entry would be a part of that package.

Reply:
datamover package will also have this entry or only S3 package will have this?

We need this entry in datamover package.

Reply:
do we need the privsep for nfs as well ?

Reply:
yes path related activities we are performing via privsep. https://github.com/trilioData/contego/blob/master/contego/privsep/path.py

Reply:
Resolving this one, as we are using system's privsep helper and its suders permission are already set by nova. I ll check in the contego installation script as well.